### PR TITLE
Onboarding Form Steps 1 and 3 Logic

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Implements form logic for steps 1 and 3 of onboarding form.

Notes: 
An error message appears stating that all of the fields aren't complete at the end of the video provided below. However, all of the fields are completed. I didn't see an error response in my network. In addition, the regular sidebar appears a few seconds after the error message appears. Could this be happening because of the check on `line 46` of step 4's form?

Fixes SC-20564 & SC-20567

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/20605256?key=2010f92cefbd9cfc2f4b038b54207bda

### Definition of Done

- [x] I have manually tested the change running it locally (having rebuilt all containers) or via unit tests 
- [ ] I have added unit and/or integration tests that cover my changes
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have updated the dependencies list if necessary (including updating yarn.lock and/or go.sum)
- [ ] I have recompiled and included new protocol buffers to reflect changes I made if necessary
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x] I have notified the reviewer via Shortcut or Slack that this is ready for review
- [ ] Front-end: Checked sm, md, lg screen resolutions for effective responsiveness
- [ ] Backend-end: Documented service configuration changes or created related devops stories

### Reviewer(s) checklist

- [ ] Front-end: I've reviewed the Figma design and confirmed that changes match the spec.
- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/features/onboarding/components/steps/organization/index.tsx [hunk 1]
```diff
@@ -1,19 +1,41 @@
...
-    console.log(values);
+    const payload = {
+      memberID: member?.id,
+      payload: {
+        ...getOnboardingStepsData(member),
+        organization: values.organization,
+      },
+    };
+    updateMember(payload);
     increaseStep();
```
Comment (masskoder) [inline web/beacon-app/src/features/onboarding/components/steps/organization/index.tsx:29]: Would you mind removing this?  We should only increase it after the request succeeds. If possible, let's remove it wherever it has been added.

**Review by masskoder — APPROVED** [review]
LGTM

**Comment by masskoder** [comment]
> ### Scope of changes
> Implements form logic for steps 1 and 3 of onboarding form.
> 
> Notes: An error message appears stating that all of the fields aren't complete at the end of the video provided below. However, all of the fields are completed. I didn't see an error response in my network. In addition, the regular sidebar appears a few seconds after the error message appears. Could this be happening because of the check on `line 46` of step 4's form?

The regular sidebar appears because the member status changes from Onboarding to Active. 
> 
> Fixes [SC-20564](https://app.shortcut.com/rotational/story/20564) & [SC-20567](https://app.shortcut.com/rotational/story/20567)
> 
> ### Type of change
> * [x]  new feature
> * [ ]  bug fix
> * [ ]  documentation
> * [ ]  testing
> * [ ]  technical debt
> * [ ]  other (describe)
> 
> ### Acceptance criteria
> https://www.awesomescreenshot.com/video/20605256?key=2010f92cefbd9cfc2f4b038b54207bda
> 
> ### Definition of Done
> * [x]  I have manually tested the change running it locally (having rebuilt all containers) or via unit tests
> * [ ]  I have added unit and/or integration tests that cover my changes
> * [ ]  I have added new test fixtures as needed to support added tests
> * [ ]  I have updated the dependencies list if necessary (including updating yarn.lock and/or go.sum)
> * [ ]  I have recompiled and included new protocol buffers to reflect changes I made if necessary
> * [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
> * [x]  I have notified the reviewer via Shortcut or Slack that this is ready for review
> * [ ]  Front-end: Checked sm, md, lg screen resolutions for effective responsiveness
> * [ ]  Backend-end: Documented service configuration changes or created related devops stories
> 
> ### Reviewer(s) checklist
> * [ ]  Front-end: I've reviewed the Figma design and confirmed that changes match the spec.
> * [ ]  Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
> * [ ]  Are there any TODOs in this PR that should be turned into stories?

File: web/beacon-app/src/features/onboarding/components/steps/organization/index.tsx (hunk 1 above)
Comment (daniellemaxwell) [reply web/beacon-app/src/features/onboarding/components/steps/organization/index.tsx]: Sure. I believe this was there before I merged changes and it got missed while making updates.

**Comment by daniellemaxwell** [comment]
@masskoder In the video, you should see that an error message appears stating that all of the fields aren't complete. However, they are. The dashboard changes after the status updates, but the user has to click `Next` again to be navigated to the completion step. The issue is that the user shouldn't see an error message if all of the fields are complete. Also, they shouldn't have to click `Next` twice if they have completed all of the required fields.

**Pull request merged**